lib/ex_doc/translations.ex

```diff
@@ -0,0 +1,401 @@
+defmodule ExDoc.Translations do
+  @locale_spanish_international "es"
+  @locale_spanish_spain "es-ES"
+  @locale_galician_spain "gl-ES"
+
+  def translate(config, text), do: t(config.language, text)
+
+  def t(locale, text) when is_atom(text),
+    do: t(locale, Atom.to_string(text))
+
+  def t(@locale_spanish_international, "Anchor for this section"), do: "Ancla para esta sección"
```

Comment:
I dont rhink we should have translate/2 and t/2. I think we should go for the latter. I think it is quiet common in software to name the translating function t to make code more readable and avoid repetitive noise